packages/hoppscotch-common/locales/en.json

```diff
@@ -568,7 +568,9 @@
     "generated_code": "Generated code",
     "go_to_authorization_tab": "Go to Authorization tab",
     "go_to_body_tab": "Go to Body tab",
+    "graphql_placeholder": "Enter a URL",
     "header_list": "Header List",
+    "http_placeholder":"Enter a URL or cURL command",
```

Comment:
Can we rename the placeholder to `Enter a URL or Paste cURL Command`? Because many users don't know that we can directly paste the cURL command. So, explicitly mentioning that would help users

Reply:
Wow. I don't know about this feature. It's cool. Agree with @anwarulislam